machines/Makefile.include

```diff
@@ -0,0 +1,13 @@
+include arch_filelist.mk
+include sim_filelist.mk
+include tb_filelist.mk
+
+VDEFINES  := $(ARCH_VDEFINES) $(SIM_VDEFINES) $(TB_VDEFINES)
+VSOURCES  := $(ARCH_VSOURCES) $(SIM_VSOURCES) $(TB_VSOURCES)
+VINCDIRS  := $(ARCH_VINCDIRS) $(SIM_VINCDIRS) $(TB_VINCDIRS)
+VINCLUDES := $(foreach dir, $(VINCDIRS), $(shell find $(dir) -name '*.vh'))
```

Comment:
Why don't we use VINCLUDES for directories (Since INCLUDES / CINCLUDES is usually directories) and VINCSRCS for .vh files.

I think you can also have VINCLUDES as a prerequisite, and then when a file in that directory changes the makefile will rebuild.

Reply:
Oh, didn't realize we can have dirs in prerequisites! Then yes, it make sense to only have `VINCLUDES` listing incidrs.